common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch6/ElasticSearchClient.scala

```diff
@@ -20,6 +20,8 @@ trait ElasticSearchClient {
 
   def port: Int
 
+  def protocol: String
```

Comment:
Shall we just combine all three into a single `url` parameter for ES6? Definitely would be a breaking change though so maybe best as a separate PR